events/registration/models.py

```diff
@@ -0,0 +1,71 @@
+import uuid
+
+from django.db import models
+from django.utils import timezone
+from django.contrib.auth.models import User as BaseUser
+from django.core.exceptions import PermissionDenied
+
+from datetime import datetime, timedelta
+
+INVITE_DAYS_TTL = 7
+
+
+class User(BaseUser):
+
+    @classmethod
+    def get_all_users(cls):
+        return cls.objects.all()
+
+    @classmethod
+    def get_user_by_id(cls, id):
+        try:
+            user = cls.objects.get(pk=id)
+        except cls.DoesNotExist:
+            return None
+        else:
+            return user
+
+
+class RegistrationConfirm(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    hash_code = models.CharField(max_length=32, unique=True)
```

Comment:
This field creation instruction ideal should look like this:
`hash_code = models.UUIDField(default=uuid.uuid4, unique=True)`
No need to specify length then - ORM handles it automatically.
https://docs.djangoproject.com/en/1.10/ref/models/fields/#uuidfield